packages/design-system-docs/src/pages/components/Autocomplete/_Autocomplete.docs.scss

```diff
@@ -18,6 +18,52 @@ The `Autocomplete` component is a parent component that adds autocomplete functi
 Style guide: components.autocomplete.react
 */
 
+/*
+<h4>Autocomplete item object</h4>
+
+<table class="ds-c-table ds-c-table--compact">
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code class="ds-u-font-weight--bold">id</code></td>
+      <td><code>string</code></td>
+      <td>
+        <p>Unique identifier for this item</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code class="ds-u-font-weight--bold">name</code></td>
+      <td><code>string</code></td>
+      <td>
+        <p>Displayed value of the item. May alternatively provide a <code>children</code> value</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code class="ds-u-font-weight--bold">children</code></td>
+      <td><code>string</code></td>
+      <td>
+        <p>Additional classes to be added to the root element. Useful for adding utility classes.</p>
```

Comment:
[fix] there's a mis-match here. The description seems to be for the `className` prop which is not currently present in the docs. Please add the `className` prop and update the description for the `children` prop.